### PR TITLE
fix: crash reporting

### PR DESCRIPTION
**Pull request by johnxnguyen**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Can't build the app for device.

### Causes

Compiler complains that "CrashReporter" is missing.

### Solutions

Add "CrashReporter" framework.

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by agisilaos — APPROVED** [review]

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1376,6 +1376,8 @@
 		EECA835426F211210087ECB0 /* UIAlertControllerFeatureConfigSnapshotTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = EECA835326F211210087ECB0 /* UIAlertControllerFeatureConfigSnapshotTests.swift */; };
 		EECA90AA287449EC0024B301 /* DeveloperFlagsView.swift in Sources */ = {isa = PBXBuildFile; fileRef = EECA90A9287449EC0024B301 /* DeveloperFlagsView.swift */; };
 		EECA90AC28744A060024B301 /* DeveloperFlagsViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = EECA90AB28744A060024B301 /* DeveloperFlagsViewModel.swift */; };
+		EECC5231297AC0A900177AB5 /* CrashReporter.xcframework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = EECC522F297AC0A800177AB5 /* CrashReporter.xcframework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+		EECC5232297AC0D300177AB5 /* CrashReporter.xcframework in Frameworks */ = {isa = PBXBuildFile; fileRef = EECC522F297AC0A800177AB5 /* CrashReporter.xcframework */; };
```
Comment (netbe) [inline Wire-iOS.xcodeproj/project.pbxproj:1380]: should be optional as datadog

**Comment by johnxnguyen** [comment]
> @johnxnguyen we should add it to the datadog-conditional script to be removed if datadog not in

Yes, I updated the script.

File: Wire-iOS.xcodeproj/project.pbxproj (hunk 1 above)
Comment (netbe) [reply Wire-iOS.xcodeproj/project.pbxproj:1380]: the linking of crashreporter should be optional in case it's not included

Comment (johnxnguyen) [reply Wire-iOS.xcodeproj/project.pbxproj]: fixed

**Review by netbe — APPROVED** [review]

**Pull request merged**